pkg/target/kusttarget.go

```diff
@@ -87,6 +90,15 @@ func unmarshal(y []byte, o interface{}) error {
 	return dec.Decode(o)
 }
 
+// makeTransformerConfig returns a complete TransformerConfig object from either files
+// or the default configs
+func makeTransformerconfig(ldr ifc.Loader, paths []string) (*config.TransformerConfig, error) {
```

Comment:
nit: casing mismatch (should be `makeTransformerConfig`)

Reply:
fixed